src/corona_hakab_model/supervisor.py

```diff
@@ -119,6 +119,44 @@ def plot(self, max_scale=True, auto_show=True, save=True):
         if auto_show:
             plt.show()
 
+    @staticmethod
+    def static_plot(simulations_info: Sequence[("SimulationManager", str, Sequence[str])], title="comparing",
```

Comment:
If we want to merge plots, we need some better architecture, not something ad-hoc like this